web/info/src/app.html

```diff
@@ -4,6 +4,22 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%sveltekit.assets%/favicon.png" />
     <meta name="viewport" content="width=device-width" />
+
+    <script>
+      // Prevent FOUC
+      if (
+        localStorage.theme === 'dark' ||
+        (!('theme' in localStorage) &&
+          window.matchMedia('(prefers-color-scheme: dark)').matches)
+      ) {
+        document.documentElement.classList.add('dark');
```

Comment:
This should be done thru `<svelte:head use:classList={...} />`, JS shouldn't be used inside the app.html template - https://svelte.dev/tutorial/svelte-head

Reply:
Thanks!